R/pkg/R/DataFrame.R

```diff
@@ -2199,3 +2199,97 @@ setMethod("coltypes",
 
             rTypes
           })
+
+#' Display the structure of a DataFrame, including column names, column types, as well as a
```

Comment:
It's ok since he has `@name` and `@title` tag below.
Apparently this is the doc style adopted in DataFrame.R - we should make it consistent across source files though at some point.